### PR TITLE
chore: disable flaky reporter snapshots for retry tests

### PR DESCRIPTION
**Pull request by astone123**
These snapshots flake a lot of the time. I don't think that snapshots are useful unless they are consistent. This PR removes them so that we can rely on our Percy step to surface legitimate UI changes.

### CODE REVIEW

**Comment by marktnoonan** [comment]
Updated the branch to trigger CI again since we had some failing tests. I think the way to review this is to understand what coverage is lost but we need a green CI run for that 😅.

I suspect this is fine. If I think we are losing too much, I'll make suggestions around how to stabilize the DOM or even remove the dynamic parts.

**Comment by tbiethman** [comment]
For reference, these tests were originally located within the old `runner` package prior to v10. We wanted snapshots of these states to have an apples-to-apples comparison between the v9/v10 command log integrations.

At this point, I don't have much of an opinion on these specific snapshots. The tests themselves are performing a lot of assertions, and we've proven a lot of it out in the last year and change. However, I anticipate these may be the only snapshots we have of attempts/retries in the command log, and it might be worth identifying one or two tests that don't flake and continuing to snapshot those.

**Comment by astone123** [comment]
@tbiethman I'd like to look into that at some point... the goal of this is just to get the Percy step to actually pass and fail meaningfully

File: packages/app/cypress/e2e/runner/retries.ui.cy.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { loadSpec } from './support/spec-loader'
-import { snapshotReporter } from './support/snapshot-reporter'
+// import { snapshotReporter } from './support/snapshot-reporter'
```
Comment (AtofStryker) [inline packages/app/cypress/e2e/runner/retries.ui.cy.ts:2]: wonder if we should just remove the commented out code. Do we know how the flake and what area of the snapshot is causing the issue?

**Pull request closed without merging**